extensions/vi-mode/tests/visual.lisp

```diff
@@ -77,14 +77,20 @@
         (cmd "v3iw")
         (ok (buf= #?"<foo ba[r]>\n"))))
     (testing "va\""
+        (skip "Skipped temporarily...")
```

Comment:
Particularly, I would create a new issue only about fixing this skipped tests and attach the issue id or even the link here